artemis-server/src/main/java/org/apache/activemq/artemis/core/deployers/impl/FileConfigurationParser.java

```diff
@@ -1456,6 +1458,10 @@ protected Pair<String, AddressSettings> parseAddressSettings(final Node node) {
             addressSettings.setEnableMetrics(XMLUtil.parseBoolean(child));
          } else if (ENABLE_INGRESS_TIMESTAMP.equalsIgnoreCase(name)) {
             addressSettings.setEnableIngressTimestamp(XMLUtil.parseBoolean(child));
+         } else if (ID_CACHE_SIZE.equalsIgnoreCase(name)) {
+            int idCacheSize = XMLUtil.parseInt(child);
+            Validators.GT_ZERO.validate(ID_CACHE_SIZE, idCacheSize);
```

Comment:
I wonder if this should use `Validators.GE_ZERO` so that users can disable the cache for specific addresses even if a "parent" address-setting uses a value > 0.

Reply:
Yeah makes sense! changed to `Validators.GE_ZERO`